src/ansys/aedt/core/visualization/post/fields_calculator.py

```diff
@@ -581,6 +625,179 @@ def calculator_write(self, expression, output_file, setup=None, intrinsics=None)
         self.ofieldsreporter.CalcStack("clear")
         return True
 
+    @pyaedt_function_handler()
+    def export(
+        self,
+        quantity,
+        solution=None,
+        variations=None,
+        output_file=None,
+        intrinsics=None,
+        phase=None,
+        sample_points=None,
+        export_with_sample_points=True,
+        reference_coordinate_system="Global",
+        export_in_si_system=True,
+        export_field_in_reference=True,
+        grid_type=None,
+        grid_center=None,
+        grid_start=None,
+        grid_stop=None,
+        grid_step=None,
+        is_vector=False,
+        assignment="AllObjects",
+        objects_type="Vol",
+    ):
+        """Export the field quantity in the top register to a file, mapping it to a grid of points.
+
+        Two options are available for defining the grid points on which to export:
+        -   Input grid points from file : Maps the field quantity to a customized grid of points.
+                                          Before using this command, you must create a file containing the points
+                                          and units.
+        -   Calculate grid points : Maps the field quantity to a three-dimensional Cartesian grid.
+                                    You specify the dimensions and spacing of the grid in the x, y, and z directions,
+                                    with units. The initial units are taken from the model.
+                                    Other grid options are: Cylindrical, in which case rho, phi and z directions must be
+                                    specified, or Spherical, in which case r, theta and phi directions must be
+                                    specified.
+
+        If you want to adopt the first option you must provide either the file containing the grid of points
+        or provide a list of sample points in ``sample_points``. In the latter case, a new file is created
+        in the working directory called "temp_points.pts" that will be automatically written with the data points
+        provided and consequently imported.
```

Comment:
Why do you say "and consequently imported" ? Does that mean that you can't export with data points without changing the current data on the fly?

Reply:
@SMoraisAnsys no because when points are provided as a list they have to be written in a .pts file and then imported by the method. Hopefully I answered your question :)